stellarphot/utils/magnitude_system_transforms.py

```diff
@@ -214,3 +214,85 @@ def load(cls) -> "PanStarrs1ToJohnsonCousins":
         # Load the transformation from a file
         path = Path(get_pkg_data_path("data/PS1_to_JC.json"))
         return cls.model_validate_json(path.read_text())
+
+
+def transform_apass_bands(table):
+    """
+    A function for transforming from the native APASS bassbands to Johnson/Cousins
+    R and I bands.
+
+    Parameters
+    ----------
+    table : `astropy.table.Table`
+        Table of catalog information. This is assumed to be in "passband" format
+    """
+    # Putting this here to avoid a circular import
+    from .magnitude_transforms import filter_transform
+
+    table["mag_RC"] = filter_transform(
+        table,
+        output_filter="R",
+        g="mag_SG",
+        r="mag_SR",
+        i="mag_SI",
+        transform="jester",
+    )
+    table["mag_IC"] = filter_transform(
+        table,
+        output_filter="I",
+        g="mag_SG",
+        r="mag_SR",
+        i="mag_SI",
+        transform="jester",
+    )
+
+    # Yes, this is dumb. You fix it if you want it less dumb.
```

Comment:
Why is this necessary? It might be better to explain the problem here in the comments if you want to give a future 'fixer' of the bug motivation.